### ISSUE TITLE
Data search capability

### ISSUE THREAD

**Issue by AFgh24** [stoewer/parquet-cli#10]
Hello
It is a good tool.
It will be great if you add the possibility of data search to the tool

**Comment by stoewer** [comment]
Yes I think some search functionality could be useful. Do you have a specific use case or sub command in mind?

**Comment by AFgh24** [comment]
Search among several Parquet files with one query
Searching for a text in a specific table in multiple parquet files
